src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/ComputerVisionClient.cs

```diff
@@ -59,19 +59,6 @@ public partial class ComputerVisionClient : ServiceClient<ComputerVisionClient>,
         /// </summary>
         public ServiceClientCredentials Credentials { get; private set; }
 
-        /// <summary>
-        /// Initializes a new instance of the ComputerVisionClient class.
-        /// </summary>
-        /// <param name='httpClient'>
-        /// HttpClient to be used
-        /// </param>
-        /// <param name='disposeHttpClient'>
-        /// True: will dispose the provided httpClient on calling ComputerVisionClient.Dispose(). False: will not dispose provided httpClient</param>
-        protected ComputerVisionClient(HttpClient httpClient, bool disposeHttpClient) : base(httpClient, disposeHttpClient)
```

Comment:
This could be a breaking change. Was this code generated using generate.ps1

Reply:
It *was* generated by `generate.ps1`, and I can only presume this is a change in how autorest generates the code.
I'd have to poke around to see what may have changed.  These are the versions used:
```
AutoRest code generation utility [version: 2.0.4283; node: v8.9.3]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\cthrash.REDMOND\.autorest\@microsoft.azure_autorest-core@2.0.4302\node_modules\@microsoft.azure\autorest-core\dist' (2.0.4302)
   Loading AutoRest extension '@microsoft.azure/autorest.csharp' (~2.3.79->2.3.79)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.55->2.3.55)

2019-03-04 21:43:41 UTC
Azure-rest-api-specs repository information
GitHub fork: Azure
Branch:      master
Commit:      1e7a3ce32b5ed2ecf968507eb3388aa07fbaff81
AutoRest information
Requested version: latest
Bootstrapper version:    autorest@2.0.4283
```

Reply:
Could you try running `autorest --reset` and regenerate the code again.

Reply:
I was branching off the old psSdkJson6 branch.  An updated PR has been made, this one will be abandoned

Reply:
Please close this PR when ready and reference it in the new PR